src/components/StakingVaults/ProviderCard.tsx

```diff
@@ -75,15 +75,16 @@ export const ProviderCard: React.FC<ProviderCardProps> = ({
 
         return (
           staking.includes(e.id as OpportunityId) &&
-          walletSupportsChain({
-            chainId: e.chainId,
-            wallet,
-            isSnapInstalled,
-            checkConnectedAccountIds: chainAccountIds,
-          })
+          (!isConnected ||
```

Comment:
We don't want to narrow opportunities by wallet support since... there is no wallet here.